src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrScanner.cs

```diff
@@ -89,7 +89,7 @@ public async Task<List<string>> ScanVmr(string? baselineFilePath, CancellationTo
     protected async Task<IEnumerable<string>> GetExclusionFilters(string? repoName, string baselineFilePath)
     {
         var text = await _fileSystem.ReadAllTextAsync(baselineFilePath);
-        return text.Split(Environment.NewLine, StringSplitOptions.RemoveEmptyEntries)
+        return text.Split(new char[] { '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
How about https://learn.microsoft.com/en-us/dotnet/api/system.io.file.readalllines?view=net-7.0?

Reply:
Or let's keep this actually
With the empty lines